pkg/api/types.go

```diff
@@ -1469,8 +1469,10 @@ type NodeSystemInfo struct {
 
 // NodeStatus is information about the current status of a node.
 type NodeStatus struct {
-	// Capacity represents the available resources of a node.
+	// Capacity represents the total resources of a node.
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Allocatable represents the resources of a node that are available for scheduling.
+	Allocatable ResourceList `json:"allocatable,omitempty"`
```

Comment:
Ok. I just realized this PR doesn't actually touch types.go, so I'll send a separate PR to address these comments.